addons/OPT/macros.hpp

```diff
@@ -0,0 +1,33 @@
+#define PREFIX OPT
+#define PATH OPT
+#define MOD OPT
+
+// Define version information of this mod
+#define MAJOR 0
```

Comment:
Wir können die Makros hier verwenden oder eine pboname.h? Das wäre mir für die Benennung der pbo noch lieber, da dann die Versionsnummer in der Datei angepasst werden kann und von makePBO übernommen wird: https://community.bistudio.com/wiki/MakePbo

Reply:
Also die Makros sollten auf jeden Fall drin bleiben, da die im Logging von CLib verwendet werden.
Die Versionsnummer im PBO-Namen find ich persönlich auch relativ ungeschickt, aber wenn man sowas will, könnte man eine `pboname.h` anlegen, die so aussieht
```
#include "macros.hpp"
pboname=opt_v#VERSION.pbo
```
und dann im ersten Build-Schritt diese Datei durch den preprocessor jagen. Dann hat man beides, muss es aber nur an einer Stelle konfigurieren.

Reply:
Habe jetzt nicht vor Augen, wie der Name der PBO zusammengesetzt wird (aber die bat sind echt einfach zu lesen, dachte sowas wäre komplizierter), wollte nur vermeiden, dass man per Hand eine nNamen oder eine Versionsnummer angeben muss. Ich möchte, dass die PBO ja eine Versionsnummer nach dem Schema Major_Minor_Build oder so bekommt und dass das eben aus der makros.hpp kommt oder sonstwo, aber nicht als Argument über die kommandozeile, wo es ja jeder dann anders machen kann / oder vergisst. Idealerweise eben automatisiert

Reply:
Okay dann füge ich bei Gelegenheit noch die Geschichte mit der Versionsnummer in die Skripte mit ein